galaxy_ng/app/settings.py

```diff
@@ -447,3 +448,20 @@
 
 # https://github.com/ansible/django-ansible-base/pull/611
 RENAMED_USERNAME_PREFIX = "galaxy_"
+
+# required by django-flags
+TEMPLATES = [
+    {
+        'BACKEND': 'django.template.backends.django.DjangoTemplates',
+        'DIRS': [],
+        'APP_DIRS': True,
+        'OPTIONS': {
+            'context_processors': [
+                'django.template.context_processors.request',
+            ],
+        },
+    },
+]
```

Comment:
This is already by pulpcore, it we override we are going to break the browsable API I guess

https://github.com/pulp/pulpcore/blob/da11429a0ace2973beb3003d96a2a3071290ef52/pulpcore/app/settings.py#L131-L145

In the case we need to manipulate the setting coming from pulpcore we will need to do in the `dynaconf_hooks.py` which will run after data is loaded from pulpcore.

For now I think this can be removed.